test/Microsoft.Azure.SignalR.Management.Tests/AadAuthFacts.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Microsoft.Azure.SignalR.Management.Tests
+{
+    public class AadAuthFacts
+    {
+        [Fact(Skip = "")]
```

Comment:
How about update `appsettings.Test.json` and put it to e2e test project? Or use the mock service?

Reply:
deleted and will be moved to E2E test.